[Platform] Allow scoping generated JSON Schema to serializer groups (#2314)
symfony/ai: src/platform/CHANGELOG.md

0.11
----

>  * Add optional `$context` argument to `Contract\JsonSchema\Factory::buildProperties()` and `buildParameters()`, carried to the describers via the new `ObjectSubject::getContext()`. Passing `['serializer_groups' => ['write']]` scopes the generated JSON Schema to the given serializer groups, so structured output and tools can expose a subset of a class (e.g. an editable view) instead of every property

Author: ```suggestion
 * Add optional `$context` argument to `Contract\JsonSchema\Factory` build methods to limit schema to subset of properties
```
